Zend/zend_execute.c

```diff
@@ -902,6 +909,8 @@ static ZEND_COLD void zend_verify_missing_arg(zend_execute_data *execute_data, u
 		const char *func_name = EX(func)->common.function_name ? ZSTR_VAL(EX(func)->common.function_name) : "main";
 		zend_execute_data *ptr = EX(prev_execute_data);
 
+		zend_undef_unpassed_args(EG(current_execute_data));
+
```

Comment:
zend_undef_unpassed_args() may be called few times. Right?
but this is possible only on slow exceptional path.